packages/venia-concept/src/components/Navigation/__tests__/categoryTree.spec.js

```diff
@@ -153,10 +154,10 @@ test('renders with product data', async () => {
         </MockedProvider>
     );
 
-    await wait();
-
-    expect(root.findAllByType(Branch).length).toBe(2);
-    expect(root.findAllByType(Leaf).length).toBe(3);
+    await waitForExpect(() => {
```

Comment:
cool, I can wait for this to merge and rebase on these changes for #729 @jimbo @zetlen